graderutils/tracebackformat.py

```diff
@@ -112,13 +115,20 @@ def clean_feedback(result_groups, config):
         for result in group["testResults"]:
             # Run all cleaning tasks for traceback
             for task in config:
-                if task.get("hide_tracebacks", False):
+                if task.get("hide_tracebacks", False) or task.get("remove_sentinel", ''):
                     # This task defines that exceptions from some class must be hidden
+                    remove_sentinel = task.get("remove_sentinel", '')
+
+                    # remove_more_sentinel has been deprecated
```

Comment:
It would be relevant to explain here  that how did it work before and maybe say something about that "it's now deprecated, but it did work plaa plaa" and then continue how old behaviour is still implemented.

In addition, it would be a good idea to adopt a way to mark these parts, so they can be removed in next major release. For reference, Django project uses `RemovedInDjango40Warning`, where 40 means v4.0, see https://github.com/django/django/search?q=RemovedInDjango40Warning&unscoped_q=RemovedInDjango40Warning